torch/_inductor/lowering.py

```diff
@@ -4038,6 +4037,19 @@ def register_pointwise_numeric_ldf64(op):
 register_pointwise_numeric(aten.nextafter)
 
 
+@register_lowering(aten.erfinv,
+                   type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT)
+def erfinv_lowering(x):
+    is_cuda = decode_device(x.get_device()).type == "cuda"
+    # triton has a primitive for erfinv
+    if is_cuda:
+        return make_pointwise(lambda x: ops.erfinv(x))(x)
+
+    # CPU doesn't have a primitive for erfinv
+    # (however, one can write decomposition for the same).
+    return fallback_handler(aten.erfinv)(x)
```

Comment:
I'm curious why this doesn't work on CPU?
https://github.com/pytorch/pytorch/blob/61b6b038b02b38e0d2758d86305fdc24edf76d00/aten/src/ATen/cpu/vec/vec256/vec256_float.h#L193